### PR TITLE
Add eslint and prettier

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

**Comment by Cherry** [comment]
Can we get hard tabs please as per https://github.com/cloudflare/workers-sdk/issues/1298? 😄 The configs at https://github.com/cloudflare/next-on-pages/pull/34 should be good.

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    singleQuote: true,
+    arrowParens: 'avoid'
```
Comment (Cherry) [inline .prettierrc.js:3]:
```suggestion
    arrowParens: 'avoid',
    useTabs: true
```
😄

**Comment by dario-piotrowicz** [comment]
@Cherry I know you already looked at the prettier file, but I just wanted to double check with you 🙂 

it's almost exactly the same as the workers-sdk one: https://github.com/cloudflare/workers-sdk/blob/main/.prettierrc
it has but it adds `arrowParens: 'avoid',` and has `"singleQuote"` set to `true` instead of `false`, these are my personal preferences, let me know if we can keep them as is, or if you think we should 100% match workers-sdk I am fine to change these as well 🙂:+1:

**Review by Cherry — APPROVED** [review]
I think `singleQuote` and `arrowParens` are completely personal preference, so if you prefer one over the other, sounds good to me! 🧡

**Review by jculvey — APPROVED** [review]

**Pull request merged**